packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -31,6 +31,11 @@ const titleWrapper = isLWorAF ? {
 };
 
 const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    '& .FriendlyPlaceholderPostsItem-root:nth-of-type(3)': {
+      marginBottom: 8
+    }
+  },
```

Comment:
@Will-Howard Sure, I've added it after 3 here (since logged out users have the two always-pinned posts). That said I was paying more attention to ghost loading states on other sites and was surprised at how little the layout shifts felt noticeable to me. On mobile it's pretty inevitable since post item height depends on the post title.

Reply:
Oh sorry I mainly meant limiting the total number of rows to be about equal to what will show in the loaded page. The gap between the pinned posts and the main post is less of an issue imo.

Separately, could you create the gap by using a `placeholderCount` on the StickiedPosts component? I think that would be a more natural way to do it

Reply:
Oops yeah I should do that 🤦‍♀️  https://github.com/ForumMagnum/ForumMagnum/pull/9136